action.yml

```diff
@@ -29,7 +29,7 @@ runs:
         len=`echo $INPUT_PM | wc -c`
         if [ $len -gt 1 ]; then
           PACKAGE_MANAGER=$(echo "$INPUT_PM" | grep -o '^[^@]*')
-          VERSION=$(echo "$INPUT_PM" | grep -o '@.*' | sed 's/^@//')
+          VERSION=$(echo "$INPUT_PM" | grep -o '@.*' || true | sed 's/^@//')
```

Comment:
Ran a quick test locally and this seems to change the output of this command.

```sh
# current command:
echo "pnpm@14.7.0" | grep -o '@.*' | sed 's/^@//'
# output:
14.7.0

# PR change:
echo "pnpm@14.7.0" | grep -o '@.*' || true | sed 's/^@//'
# output:
@14.7.0
```

Grouping the central command does seem to work though:
```suggestion
          VERSION=$(echo "$INPUT_PM" | (grep -o '@.*' || true) | sed 's/^@//')
```
(Although I am 100% not a shell script expert so this may not be the correct fix)

Reply:
Oh, you're right
my command is equal to:
echo "pnpm@14.7.0" | grep -o '@.*' || { true | sed 's/^@//'; }
The proper command will be 
echo "pnpm@14.7.0" | { grep -o '@.*' ||  true; } | sed 's/^@//'